spec/solidus_content/providers/dato_cms_spec.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusContent::Providers::DatoCms do
+  describe '.entry_type_fields' do
+    subject { described_class.entry_type_fields }
+
+    it { is_expected.to eq(%i[api_token environment]) }
+  end
+
+  describe '.entry_fields' do
+    subject { described_class.entry_fields }
+
+    it { is_expected.to eq(%i[item_id version]) }
+  end
+
+  describe '.call' do
+    subject do
+      described_class.call(
+        type_options: {
+          api_token: api_token,
+          environment: environment
+        },
+        options: {
+          item_id: item_id,
+          version: version
+        }
+      )
+    end
+
+    let(:api_token) { 'api_token' }
+    let(:environment) { 'environment' }
+    let(:item_id) { 'item_id' }
+    let(:version) { 'version' }
+
+    let(:repo) do
+      object_double(
+        Dato::Site::Client.new(api_token, environment: environment).items,
+        find: fields
+      )
+    end
+    let(:fields) { instance_double(HashWithIndifferentAccess) }
+    let(:dato) { object_double(Dato::Site::Client.new(api_token, environment: environment)) }
+
+    before do
+      repo # This has to created first!
```

Comment:
Any reason for not using `let!(:repo)` instead? It should be equivalent from what I see.

Reply:
Years of rubocop-rspec warnings, maybe :D Hahaha! No problem, I can change it to `let!` :)

Reply:
Done!

Reply:
> Years of rubocop-rspec warnings

🤦‍♂️ 🤖 hahahaha 😄